ddtrace/internal/_encoding.pyx

```diff
@@ -91,6 +89,12 @@ cdef class Packer(object):
         PyMem_Free(self.pk.buf)
         self.pk.buf = NULL
 
+    cpdef _flush_buffer(self):
```

Comment:
If we don't expect this to be of use outside of the `Packer` class then we could make it a `cdef inline`.

Reply:
This would be a type `char*` right?

Reply:
I think this would have to be `object`